docs/administration.projects.md

```diff
@@ -28,42 +28,40 @@ To edit the quotas, please follow these steps:
 5. Also under the **Actions** column, you reach the **Quota Dashboard**. These quotas are project-specific.
 6. You can now change the quotas as desired.
 
-### Delete Projects
+### Delete Tenants
```

Comment:
q: Is this whole section maybe better fit for its own page? As it will also be its own page in the Admin Area. It feels a bit weird to me to place this in the projects page.

Reply:
This page includes all functionalities regarding projects. So I wouldn't place it completely into a new page currently. Since we also don't do it for other models. Maybe we can take it up when having the overhaul of docs.